### ISSUE TITLE
Task-oriented data conversions (e.g. households, etc.)

### ISSUE THREAD

**Issue by albertmeronyo** [CLARIAH/wp4-requirements#12]
RDF now allows us to express the data in multiple forms. This would be useful to represent some relations which are meaningful for specific tasks, e.g.:
- Household relations (example: find twins, find the oldest sibling of X, etc.)

**Comment by rlzijdeman** [comment]
Could you please elaborate @albertmeronyo ? I'm not sure what you mean?

**Comment by albertmeronyo** [comment]
I think this happened while you were presenting an example of a table containing family relationships. If I remember correctly, the way of expressing those relationships in one single table was pretty tricky, because rows have to reference other rows. I think I made this as a note to remember that we should think about how to conveniently model that in RDF (since a straightforward CSV to RDF conversion wouldn't work).

Now I wonder how much of this could be achieved by using the awesome JSON-LD templating strategy that @RinkeHoekstra is now using in e.g. https://github.com/CLARIAH/wp4-converters/commit/a2162b8181b749a848a18117b4dcc6163c71d2ad

**Comment by rijpma** [comment]
Yes, in the census-data it is quite tricky, for example, to get sibling pairs. You'd have to make a query like https://github.com/CLARIAH/wp4-queries/commit/1292e7149cb9e376e7ae133592ee4f8a1220d41a.

One possibility is to use INSERT queries (`INSERT{?son family:sister ?daugher}`, but that means it is no longer documented in one place how we went from dataset to triples and it might not be the most efficient way of doing it.

I think that in the templating you'd need to refer to multiple columns (e.g. houshold_id as well as relate) and previous rows, right? Is that possible?
